### ISSUE TITLE
Remove `--with-tzpath` setting

### ISSUE THREAD

**Issue by isuruf** [conda-forge/python-feedstock#403]
The conda location is supposed to be appended instead of overriding.

cc @ocefpaf, @mbargull

**Comment by mbargull** [comment]
Why not prepend? (And if we prepend, we could as well override as currently is.)

**Comment by isuruf** [comment]
I can't remember the reason, but there was some discussion as to what to do. cc @ocefpaf, @mingwandroid

**Comment by ocefpaf** [comment]
> Why not prepend? (And if we prepend, we could as well override as currently is.)

Yep. We could pre-pended but then that would not really make a difference b/c our path would always be present since our tzdata is a Python dependency.

The pep does suggests to append but, b/c we do have our own tzdata, overriding it not that bad. That was modeled after a build without tzdata as a dependency or as an optional dependency.
